### PR TITLE
arm64 support statement

### PR DESCRIPTION
**Pull request by stefanoochoa**

### CODE REVIEW

**Comment by stefanoochoa** [comment]
carried over from https://github.com/MicrosoftDocs/azure-docs/pull/124412, ready for merge

**Comment by Jak-MS** [comment]
@johnmarco 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by stefanoochoa** [comment]
Can we get this merged please @johnmarco? 🙂

File: articles/azure-arc/servers/prerequisites.md [hunk 1]
```diff
@@ -36,29 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
+| Windows Server                      | 2022        | ✅    | ❌    |                                                                                                  |
+
+For Windows Server, both Desktop and Server Core experiences are supported. Azure Editions are supported on Azure Stack HCI.
 
 The Azure Connected Machine agent isn't tested on operating systems hardened by the Center for Information Security (CIS) Benchmark.
 
-> [!NOTE]
-> [Azure Connected Machine agent version 1.44](agent-release-notes.md#version-144---july-2024) is the last version to officially support Debian 10, Ubuntu 16.04, and Azure Linux (CBL-Mariner) 1.0.
-> 
+## ARM64 Architecture support
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:83]:
```suggestion
## ARM64 architecture support
```

File: articles/azure-arc/servers/prerequisites.md [hunk 2]
```diff
@@ -36,29 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
+
+Not all features, virtual machine extensions, and services are supported on ARM64 at this point. Below are a few features which are known to be supported. For full details on ARM64 compatibility or to check if other services are supported, please
+refer to the documentation for the service you wish to use.
+- RunCommand
+- CustomScriptExtension
+- Azure Monitor Agent
+
+Please note that Machine Configuration is NOT compatible with ARM64 at this time.
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:91]:
```suggestion
> [!NOTE]
> 
> Machine configuration is not compatible with ARM64 at this time.
```

File: articles/azure-arc/servers/prerequisites.md [hunk 3]
```diff
@@ -36,29 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
-> [!NOTE]
-> [Azure Connected Machine agent version 1.44](agent-release-notes.md#version-144---july-2024) is the last version to officially support Debian 10, Ubuntu 16.04, and Azure Linux (CBL-Mariner) 1.0.
-> 
+## ARM64 Architecture support
+
+Not all features, virtual machine extensions, and services are supported on ARM64 at this point. Below are a few features which are known to be supported. For full details on ARM64 compatibility or to check if other services are supported, please
+refer to the documentation for the service you wish to use.
+- RunCommand
+- CustomScriptExtension
+- Azure Monitor Agent
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:89]:
```suggestion
Not all features, virtual machine extensions, and services are supported on ARM64 at this point. Below are a few features which are known to be supported. For full details on ARM64 compatibility or to check if other services are supported, refer to the documentation for the service you wish to use.

- RunCommand
- CustomScriptExtension
- Azure Monitor Agent
```

File: articles/azure-arc/servers/prerequisites.md [hunk 4]
```diff
@@ -36,29 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
-  * Azure Editions are supported on Azure Stack HCI
+Azure Arc supports the following Windows and Linux operating systems. x86-64 (64-bit) architecture is fully supported. Some features are supported on ARM64, see below. The Azure Connected Machine agent doesn't run on 32-bit architectures.
+
+| Operating System                    | Version     | x86-64 | ARM64 | Notes                                                                                            |
+| ----------------------------------- | ----------- | ------ | ----- | ------------------------------------------------------------------------------------------------ |
+| AlmaLinux                           | 9           | ✅    | ❌    |                                                                                                  |
+| Amazon Linux                        | 2           | ✅    | ❌    |                                                                                                  |
+| Amazon Linux                        | 2023        | ✅    | ✅    | see [ARM64 Architecture support](#arm64-architecture-support)                                    |
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:45]:
```suggestion
| Amazon Linux                        | 2023        | ✅    | ✅    | see [ARM64 architecture support](#arm64-architecture-support)                                    |
```

File: articles/azure-arc/servers/prerequisites.md [hunk 5]
```diff
@@ -36,29 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
-* SUSE Linux Enterprise Server (SLES) 12 SP3-SP5 and 15
-* Ubuntu 18.04, 20.04, and 22.04 LTS
-* Windows 10, 11 (see [client operating system guidance](#client-operating-system-guidance))
-* Windows IoT Enterprise
-* Windows Server 2012, 2012 R2, 2016, 2019, and 2022
-  * Both Desktop and Server Core experiences are supported
-  * Azure Editions are supported on Azure Stack HCI
+Azure Arc supports the following Windows and Linux operating systems. x86-64 (64-bit) architecture is fully supported. Some features are supported on ARM64, see below. The Azure Connected Machine agent doesn't run on 32-bit architectures.
+
+| Operating System                    | Version     | x86-64 | ARM64 | Notes                                                                                            |
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:41]:
```suggestion
| Operating system                    | Version     | x86-64 | ARM64 | Notes                                                                                            |
```

File: articles/azure-arc/servers/prerequisites.md [hunk 6]
```diff
@@ -36,29 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
+| Red Hat Enterprise Linux (RHEL)     | 8           | ✅    | ❌    |                                                                                                  |
+| Red Hat Enterprise Linux (RHEL)     | 9           | ✅    | ❌    |                                                                                                  |
+| Rocky Linux                         | 8           | ✅    | ❌    |                                                                                                  |
+| Rocky Linux                         | 9           | ✅    | ❌    |                                                                                                  |
+| SUSE Linux Enterprise Server (SLES) | 12 SP3-SP5  | ✅    | ❌    |                                                                                                  |
+| SUSE Linux Enterprise Server (SLES) | 15          | ✅    | ✅    | see [ARM64 Architecture support](#arm64-architecture-support)                                    |
+| Ubuntu                              | 16.04       | ⚠️    | ❌    | **Limited Support**, see [Limited support operating systems](#limited-support-operating-systems) |
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:65]:
```suggestion
| Ubuntu                              | 16.04       | ⚠️    | ❌    | **Limited support**, see [Limited support operating systems](#limited-support-operating-systems) |
```

File: articles/azure-arc/servers/prerequisites.md [hunk 7]
```diff
@@ -36,29 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
+| Azure Linux (CBL-Mariner)           | 1.0         | ⚠️    | ❌    | **Limited Support**, see [Limited support operating systems](#limited-support-operating-systems) |
+| Azure Linux (CBL-Mariner)           | 2.0         | ✅    | ❌    |                                                                                                  |
+| Azure Linux (CBL-Mariner)           | 3.0         | ✅    | ❌    |                                                                                                  |
+| Azure Stack HCI                     |             |       |        |                                                                                                  |
+| Centos                              | 7           | ⚠️    | ❌    | **Limited Support**, see [Limited support operating systems](#limited-support-operating-systems) |
+| Centos                              | 8           | ⚠️    | ❌    | **Limited Support**, see [Limited support operating systems](#limited-support-operating-systems) |
+| Debian                              | 10          | ⚠️    | ❌    | **Limited Support**, see [Limited support operating systems](#limited-support-operating-systems) |
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:52]:
```suggestion
| Centos                              | 7           | ⚠️    | ❌    | **Limited support**, see [Limited support operating systems](#limited-support-operating-systems) |
| Centos                              | 8           | ⚠️    | ❌    | **Limited support**, see [Limited support operating systems](#limited-support-operating-systems) |
| Debian                              | 10          | ⚠️    | ❌    | **Limited support**, see [Limited support operating systems](#limited-support-operating-systems) |
```

File: articles/azure-arc/servers/prerequisites.md [hunk 8]
```diff
@@ -36,29 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
+
+| Operating System                    | Version     | x86-64 | ARM64 | Notes                                                                                            |
+| ----------------------------------- | ----------- | ------ | ----- | ------------------------------------------------------------------------------------------------ |
+| AlmaLinux                           | 9           | ✅    | ❌    |                                                                                                  |
+| Amazon Linux                        | 2           | ✅    | ❌    |                                                                                                  |
+| Amazon Linux                        | 2023        | ✅    | ✅    | see [ARM64 Architecture support](#arm64-architecture-support)                                    |
+| Azure Linux (CBL-Mariner)           | 1.0         | ⚠️    | ❌    | **Limited Support**, see [Limited support operating systems](#limited-support-operating-systems) |
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:46]:
```suggestion
| Azure Linux (CBL-Mariner)           | 1.0         | ⚠️    | ❌    | **Limited support**, see [Limited support operating systems](#limited-support-operating-systems) |
```

File: articles/azure-arc/servers/prerequisites.md [hunk 9]
```diff
@@ -36,29 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
+| Red Hat Enterprise Linux (RHEL)     | 7           | ✅    | ❌    |                                                                                                  |
+| Red Hat Enterprise Linux (RHEL)     | 8           | ✅    | ❌    |                                                                                                  |
+| Red Hat Enterprise Linux (RHEL)     | 9           | ✅    | ❌    |                                                                                                  |
+| Rocky Linux                         | 8           | ✅    | ❌    |                                                                                                  |
+| Rocky Linux                         | 9           | ✅    | ❌    |                                                                                                  |
+| SUSE Linux Enterprise Server (SLES) | 12 SP3-SP5  | ✅    | ❌    |                                                                                                  |
+| SUSE Linux Enterprise Server (SLES) | 15          | ✅    | ✅    | see [ARM64 Architecture support](#arm64-architecture-support)                                    |
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:64]:
```suggestion
| SUSE Linux Enterprise Server (SLES) | 15          | ✅    | ✅    | see [ARM64 architecture support](#arm64-architecture-support)                                    |
```

File: articles/azure-arc/servers/prerequisites.md [hunk 10]
```diff
@@ -36,29 +36,61 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
...
+| Ubuntu                              | 18.04       | ✅    | ❌    |                                                                                                  |
+| Ubuntu                              | 20.04       | ✅    | ❌    |                                                                                                  |
+| Ubuntu                              | 22.04       | ✅    | ❌    |                                                                                                  |
+| Windows Client                      | 10          | ✅    | ❌    | see [client operating system guidance](#client-operating-system-guidance)                        |
+| Windows Client                      | 11          | ✅    | ❌    | see [client operating system guidance](#client-operating-system-guidance)                        |
+| Windows IoT Enterprise              |             |       |        |                                                                                                  |
+| Windows Server                      | 2008 R2 SP1 | ⚠️    | ❌    | **Limited Support**, see [Limited support operating systems](#limited-support-operating-systems) |
```
Comment (Jak-MS) [inline articles/azure-arc/servers/prerequisites.md:72]:
```suggestion
| Windows Server                      | 2008 R2 SP1 | ⚠️    | ❌    | **Limited support**, see [Limited support operating systems](#limited-support-operating-systems) |
```

**Pull request merged**